admin_tests/metrics/test_utils.py

```diff
@@ -25,25 +26,25 @@ class TestMetricsGetProjects(AdminTestCase):
     def setUp(self):
         super(TestMetricsGetProjects, self).setUp()
         Node.remove()
-        self.node = NodeFactory(
-            category='project', is_public=True)  # makes 2 nodes bc of category
-        self.reg = RegistrationFactory()  # makes 2 nodes
-        self.node_2 = NodeFactory()
+        self.public_node = ProjectFactory(is_public=True)
+        self.private_node = ProjectFactory(is_public=False)
+        self.node_2 = NodeFactory()  # creates parent project + node
+        self.reg = RegistrationFactory(project=self.public_node)
 
-    def test_get_all_nodes(self):
+    def test_get_all_top_level_nodes(self):
         count = get_projects()
-        nt.assert_equal(count, 5)
+        nt.assert_equal(count, 4)
 
-    def test_get_public_nodes(self):
+    def test_get_public_top_level_nodes(self):
         count = get_projects(public=True)
         nt.assert_equal(count, 1)
 
     def test_get_registrations(self):
         count = get_projects(registered=True)
         nt.assert_equal(count, 1)
 
-    def test_time(self):
-        time = self.node.date_created - timedelta(seconds=1)
+    def test_no_results_before_old_time(self):
```

Comment:
This maybe could be disambiguated a tiny bit more. Is it, "No nodes before nodes were created", essentially? "No nodes a week ago because this test should only take a few seconds to run"? :)